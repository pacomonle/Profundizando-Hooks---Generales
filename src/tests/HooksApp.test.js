const { shallow } = require("enzyme")
import React from 'react';
import HooksApp from '../HooksApp';
import '@testing-library/jest-dom'




describe('Prueba <HooksApp></HooksApp>', () => {

    test('should mostrarse correctamente', () => {
        const wrapper = shallow(<HooksApp></HooksApp>)
        expect(wrapper).toMatchSnapshot()
        
    })
    
    
})
