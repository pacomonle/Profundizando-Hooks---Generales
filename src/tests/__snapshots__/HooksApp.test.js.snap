// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba <HooksApp></HooksApp> should mostrarse correctamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
